src/plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -293,8 +298,11 @@ def __init__(
             self.det_hdir = self._default_detector_hdir()
 
         # Calculate the detector vdir
-        ny = np.cross(self.det_hdir, self.det_n)
-        self.det_vdir = -ny / np.linalg.norm(ny)
+        if detector_vdir is not None:
+            self.det_vdir = detector_vdir / np.linalg.norm(detector_vdir)
+        else:
+            ny = np.cross(self.det_hdir, self.det_n)
+            self.det_vdir = -ny / np.linalg.norm(ny)
```

Comment:
If `detector_vdir` has a default argument, then we might be able to simplify this into a single line.

Reply:
I don't think so for the same reason below: `ny` needs to be calculated on line 304 before you can do the normalization on line 305? If there was a `np.linalg.normalize` function, that would work, but I don't think there is.

Reply:
Ah, makes sense!  I remember how math works! 🙃